_schema.graphql

```diff
@@ -1121,9 +1121,10 @@ type ArtworkContextFair {
   hours: String
   href: String
   image: Image
+  is_active: Boolean @deprecated(reason: "Prefer isActive instead")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 1124;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_active`